test/test_common/utility_test.cc

```diff
@@ -27,6 +29,78 @@ TEST(headerMapEqualIgnoreOrder, NotEqual) {
   EXPECT_FALSE(TestUtility::headerMapEqualIgnoreOrder(lhs, rhs));
 }
 
+TEST(protoEqIgnoreField, ActuallyEqual) {
```

Comment:
Nit: `ProtoEqIgnoreField`

Reply:
Done, and changed the others in the file as well.